test/support/vm/openldap/README.md

```diff
@@ -27,6 +46,10 @@ $ export INTEGRATION_HOST=$ip
 
 # now run tests without having to set ENV variables
 $ time bundle exec rake
+
+# Once you're all done
+$ cd test/support/vm/openldap
+$ vagrant destroy
 ```
 
 You may need to `gem install vagrant` first in order to provision the VM.
```

Comment:
I wasn't able to run the tests according to these instructions. Here's a gist of the error https://gist.github.com/jch/48c8855edf47a58cbf953cf6adb917d0